hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -83,7 +83,7 @@ public record ContractsConfig(
         @ConfigProperty(value = "systemContract.scheduleService.signSchedule.enabled", defaultValue = "true")
                 @NetworkProperty
                 boolean systemContractSignScheduleEnabled,
-        @ConfigProperty(value = "systemContract.scheduleService.authorizeSchedule.enabled", defaultValue = "false")
+        @ConfigProperty(value = "systemContract.scheduleService.authorizeSchedule.enabled", defaultValue = "true")
```

Comment:
The issue says just increase test coverage. But we are enabling this feature right ?